src/LowLevel/SecurityToken.ts

```diff
@@ -205,6 +237,26 @@ export class SecurityToken extends Contract<SecurityTokenContract> {
     return addresses.map(address => new UsdTieredSto({ address, context }));
   }
 
+  /**
+   * Given STO module type and address, this function will return a generic, LowLevel Sto object.
+   *
+   * @TODO remon-nashid: shouldn't module address be enough to fetch a module?
+   */
+  public getSTOModule = async ({ address, stoType }: GetSTOModuleArgs): Promise<Sto | null> => {
+    const { context } = this;
+    const { methods } = this.contract;
+
+    const { 3: isArchived } = await methods.getModule(address).call();
+    if (isArchived) return null;
+
+    if (stoType === StoModuleTypes.Capped) {
+      return new CappedSto({ address, context });
+    } else if (stoType === StoModuleTypes.UsdTiered) {
+      return new UsdTieredSto({ address, context });
+    }
```

Comment:
You should throw an error if the `stoType` doesn't match any of the supported types